### PR TITLE
Update traefik to 1.2.3

### PR DESCRIPTION
**Pull request by toyorg**

### CODE REVIEW

File: templates/traefik/6/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  image: rawmind/alpine-traefik:1.2.3-1
+  environment:
+  - CONF_INTERVAL=${refresh_interval}
+  - TRAEFIK_HTTP_PORT=${http_port}
+  - TRAEFIK_HTTPS_PORT=${https_port}
+  - TRAEFIK_HTTPS_ENABLE=${https_enable}
+  - TRAEFIK_ACME_ENABLE=${acme_enable}
+  - TRAEFIK_ACME_EMAIL=${acme_email}
+  - TRAEFIK_ACME_ONDEMAND=${acme_ondemand}
+  - TRAEFIK_ACME_ONHOSTRULE=${acme_onhostrule}
```
Comment (rawmind0) [inline templates/traefik/6/docker-compose.yml]: add env var to control InsecureSkipVerify param
```
  - TRAEFIK_INSECURE_SKIP=${insecure_skip}
```

File: templates/traefik/6/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      label: "SSL key"
+      type: "multiline"
+      required: false
+      default: ""
+    - variable: "ssl_crt"
+      description: "SSL cert to secure the service. *Required if you enable https"
+      label: "SSL crt"
+      type: "multiline"
+      required: false
+      default: ""
```
Comment (rawmind0) [inline templates/traefik/6/rancher-compose.yml]: Add catalog question for InsecureSkipVerify param
```
    - variable: "insecure_skip"
      description: "Enable InsecureSkipVerify param."
      label: "InsecureSkipVerify:"
      required: true
      default: false 
      type: "boolean"
```

File: templates/traefik/6/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+- https_enable = <false | true | only>
+  - false: Enable http enpoints and disable https ones.
+  - true: Enable http and https endpoints.
+  - only: Enable https endpoints and redirect http to https.
+- acme_enable = false               # Enable/Disable acme traefik support.
+- acme_email = "test@traefik.io"    # acme user email
+- acme_ondemand = true              # acme ondemand parameter.
+- acme_onhostrule = true            # acme onHostRule parameter.
+- ssl_key # Paste your ssl key. *Required if you enable https
+- ssl_crt # Paste your ssl crt. *Required if you enable https
```
Comment (rawmind0) [inline templates/traefik/6/README.md]: Add comment about InsecureSkipVerify param
```
- insecure_skip = false	  # Enable InsecureSkipVerify param.
```

**Comment by rawmind0** [comment]
Hi @toyorg ... could you please add last changes to  be able to set InsecureSkipVerify param??

**Comment by toyorg** [comment]
@rawmind0 just added!

**Comment by rawmind0** [comment]
Cool..thanks @toyorg .. :)

LGTM

**Pull request merged**